exonum-java-binding/core/src/test/java/com/exonum/binding/core/runtime/ServiceRuntimeConfigurationIntegrationTest.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.binding.core.runtime;
+
+import static java.util.Collections.emptyMap;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import com.exonum.binding.test.RequiresNativeLibrary;
+import com.exonum.binding.test.runtime.ServiceArtifactBuilder;
+import com.google.inject.Guice;
+import com.google.inject.Injector;
+import com.google.inject.Stage;
+import java.io.IOException;
+import java.nio.file.Path;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * An integration test of the runtime: the runtime module bindings and ServiceRuntime integration
+ * with its dependencies.
+ */
+@RequiresNativeLibrary
+class ServiceRuntimeConfigurationIntegrationTest {
+
+  private static final ServiceArtifactId ARTIFACT_ID =
+      ServiceArtifactId.parseFrom("com.exonum.binding:test-service:1.0.0");
+  private static final String ARTIFACT_FILENAME = "test-service.jar";
+
+  @BeforeEach
+  void createValidArtifact(@TempDir Path tmpArtifactDir) throws IOException {
+    Class<?> serviceModule = TestServiceModule.class;
```

Comment:
Why separate variable?

Reply:
Inlined, thanks, probably a refactoring artifact, this method used to be parameterized.